localization/english/bookmark/bookmark_508_l_english.yml

```diff
@@ -1,11 +1,10 @@
 ﻿l_english:
  bm_the_gnoll_war:0 "Gnoll War"
- bm_the_gnoll_war_desc:0 "PH"
-
+ bm_the_gnoll_war_desc:0 "After the collapse of Arathor, humanity divided into several kingdoms and city-states. Settlers from Arathi migrated to the southern land of Azeroth and founded the city of Stormwind. Led by the noble Wrynn dynasty, the kingdom became a large and prosperous nation. The new king Barathen now has to face the rise of another ambitious ruler, Garfang, leader of the largest gnoll pack ever seen and determined to conquer and plunder the rich human lands.\n\nWill Stormwind maintain its place as the dominant power of the sub-continent or is it time for the mongrel gnolls to finally rule?"
 
  #Barathen Wrynn
  barathen_wrynn_character:0 "King Barathen"
- barathen_wrynn_character_desc:0 "PH"
+ barathen_wrynn_character_desc:0 "After the death of his father, Landan Wrynn, king Barathen now has to rule during the gravest crisis the kingdom faced since its foundation, but the king remains adamant. With strong will, discipline and the superiority of his tactics, the king is confident that he can defeat his barbaric foes, paving the way for Stormwind to become the mightiest of all human nations."
```

Comment:
```suggestion
 barathen_wrynn_character_desc:0 "After the death of his father, Landan Wrynn, king Barathen now has to rule during the gravest crisis the kingdom faced since its foundation, but the king remains adamant. With a strong will, discipline, and the superiority of his tactics, the king is confident that he can defeat his barbaric foes, paving the way for Stormwind to become the mightiest of all human nations."
```